### DISCUSSION TITLE
[Feature] go-to-date from explore and memories

### DISCUSSION THREAD

**Discussion by claranet-retailweb** [immich-app/immich#5858] [Feature Request]
### The feature

The feature would be an option in the menu of a picture shown in search mode or memory mode, that would allow user to click "go to this day" and be forwarded in the timeline to the timestamp of the shown picture. 

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by lythandas** [comment]
Sorry, this discussion was created using my work account, I will follow-up with my personnal account

**Comment by memehammad** [comment]
Bump! This feels like an essential feature missing from immich and even "competitors" like Google photos. It's so annoying that when you're for example searching for "Birthday" you find the day you're looking for, have to write down the date, then either use the fiddly timeline slider thingy to go to that date or search again. 
Please add this! ♥️

**Comment by bo0tzz** [comment]
#4115
